src/RefractAPI.cc

```diff
@@ -87,7 +104,7 @@ namespace drafter {
         return element;
     }
 
-    refract::IElement* MetadataToRefract(const NodeInfo<snowcrash::Metadata>& metadata)
+    refract::IElement* MetadataToRefract(const NodeInfo<snowcrash::Metadata>& metadata, ConversionContext& context)
```

Comment:
We need it here since we are using it at https://github.com/apiaryio/drafter/pull/295/files#diff-2ab1b5da6ee8cb5b29287e4ebfd30e94R538